packages/shared/src/server/otel/OtelIngestionProcessor.ts

```diff
@@ -1362,7 +1366,10 @@ export class OtelIngestionProcessor {
     }
 
     // LiveKit
-    input = attributes["lk.input_text"];
+    input =
+      attributes["lk.input_text"] ??
+      attributes["lk.user_transcript"] ??
+      attributes["lk.chat_ctx"];
```

Comment:
Would you want to map `lk.function_tool.arguments` here as well since you list it above?